app/controllers/registration_controller.rb

```diff
@@ -0,0 +1,26 @@
+class RegistrationController < ApplicationController
+  skip_before_action :require_login
+
+  def index
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      auto_login(@user)
+      redirect_to projects_path
+    else
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.